src/Take.Blip.Builder/InputReplyHander.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Lime.Messaging.Contents;
+using Lime.Protocol;
+using Take.Blip.Builder;
+using Take.Blip.Builder.Models;
+/// <summary>
+/// Class responsible for handling input reply messages.
+/// </summary>
+public class InputReplyHandler : IInputMessageHandler
+{
+    /// <summary>
+    /// Validates a reply message and extracts the replied content.
+    /// </summary>
+    /// <param name="message">The input message to be validated.</param>
+    /// <returns>
+    /// A tuple containing a boolean value indicating the validation result
+    /// and the modified message with the replied content.
+    /// </returns>
+    public (bool MessageHasChanged, Message NewMessage) HandleMessage(Message message)
+    {
+        if (message.Content is Reply reply)
+        {
+            message.Content = reply.Replied?.Value;
```

Comment:
message.Content can't be null because may raise errors
eg.: line 154 of FlowManager "Input = message.Content.ToString()"

Reply:
In this scenario, what do you suggest? Should I assign a different value than null to the message content?

Reply:
It makes sense for this validation to be placed inside the InputReplyHandler class. If the message content is null, it will not be considered a Reply message type.

Reply:
From what I understood, you meant that reply.Replied cannot be null, not the message content. Based on that, I made the correction by adding an empty PlainText if replied is null.